domains/event-store/poc/EventStore/EventConsumer.cs

```diff
@@ -0,0 +1,7 @@
+using System;
+
+namespace EventStore;
+
+public interface IEventConsumer<T> { // can contain re-ordering mechanisms based on `Event.IssuedFraction`
```

Comment:
There is a mismatch between name of interface and file name. File name should be "IEventConsumer"